### ISSUE TITLE
Set velocities only explicitly in POSITION_TARGET_LOCAL_NED

### ISSUE THREAD

**Issue by kripper** [mavlink/mavlink#1966]
My understanding is that:
- `POSITION_TARGET_LOCAL_NED` can be used to set the vehicle velocities
- setting a target position and velocities at the same time is incompatible (EDIT: it's not incompatible)

If we want to set `vx`, `vy `and `vz `to zero, I think it is counterintuitive to set all other `POSITION_TARGET_TYPEMASK` to `1` except the velocities mask (suposing that this is the way to tell that we want to set velocities only).

Besides, if in the future new masking flags are added, we would have to change the code to ignore new flags.

Instead of "ignore bits", we should have "set bits", or better a bit to tell explicitely that we are setting velocities.

Please consider this when there is a chance (MAVLink 3).

**Comment by hamishwillee** [comment]
Yes, in some systems you can set positions and velocities at the same time - velocity is used as feedforward. For example, see PX4 docs here: https://docs.px4.io/main/en/flight_modes/offboard.html#mavlink-messages
